.github/workflows/Tests.yml

```diff
@@ -34,6 +34,7 @@ jobs:
           - "FIRK(EXPANDED)"
           - "FIRK(NESTED)"
           - "WRAPPERS"
+        julia-version: [1.10]
```

Comment:
Not sure if this is the correct way to fall back to Julia 1.10? Seems the CI is still testing on 1.11.